.gitignore

```diff
@@ -0,0 +1,2 @@
+node_modules
+package-lock.json
```

Comment:
Package lock should be commited: https://docs.npmjs.com/files/package-lock.json

Please remove that.